docs/source/conf.py

```diff
@@ -23,8 +23,23 @@
 import torch
 import torchvision
 import pytorch_sphinx_theme
+from sphinxcontrib import googleanalytics
 
 
+# Wrap sphinxcontrib-googleanalytics setup() function to avoid a Sphinx warning:
+# "WARNING: extension ‘sphinxcontrib.googleanalytics’ returned an unsupported
+# object from its setup() function; it should return None or a metadata
+# dictionary"
+_googleanalytics_setup_original = googleanalytics.setup
+
+
+def _googleanalytics_setup_wrapper(app):
+    _googleanalytics_setup_original(app)
+    return {"version": "0.1"}
+
+
+googleanalytics.setup = _googleanalytics_setup_wrapper
```

Comment:
@mattip @fmassa Do we typically add Google Analytics in the documentation?

Reply:
It is up to the project. I see google analytics on the pytorch doc site, I imagine this is part of the theme.

Reply:
I'm ok adding Google Analytics to the doc